crates/brioche-core/src/project.rs

```diff
@@ -968,9 +968,7 @@ fn project_lockfile(project: &Project) -> Lockfile {
     let mut git_refs = BTreeMap::new();
     for (static_, output) in project.statics.values().flatten() {
         match static_ {
-            StaticQuery::Include(_) | StaticQuery::Glob { .. } => {
-                continue;
-            }
+            StaticQuery::Include(_) | StaticQuery::Glob { .. } => {}
```

Comment:
There is no need for a continue statement here, since there is nothing after the match (https://rust-lang.github.io/rust-clippy/rust-1.86.0/index.html#needless_continue)